docs/examples.rst

```diff
@@ -0,0 +1,215 @@
+Examples
+========
+
+Look at the `complete examples <https://github.com/rhinstaller/dasbus/tree/master/examples>`_ or
+`DBus services <https://github.com/rhinstaller/anaconda/tree/master/pyanaconda/modules>`_ of
+the Anaconda Installer for more inspiration.
+
+Basic usage
+-----------
+
+Show the current hostname.
+
+.. code-block:: python
+
+    from dasbus.connection import SystemMessageBus
+    bus = SystemMessageBus()
+
+    proxy = bus.get_proxy(
+        "org.freedesktop.hostname1",
+        "/org/freedesktop/hostname1"
+    )
+
+    print(proxy.Hostname)
+
+Send a notification to the notification server.
+
+.. code-block:: python
+
+    from dasbus.connection import SessionMessageBus
+    bus = SessionMessageBus()
+
+    proxy = bus.get_proxy(
+        "org.freedesktop.Notifications",
+        "/org/freedesktop/Notifications"
+    )
+
+    id = proxy.Notify(
+        "", 0, "face-smile", "Hello World!",
+        "This notification can be ignored.",
+        [], {}, 0
+    )
+
+    print("The notification {} was send.".format(id))
```

Comment:
"sent"

Reply:
Fixed in https://github.com/rhinstaller/dasbus/pull/29.